src/compare.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+import fsExtra from 'fs-extra';
+import {PNG} from 'pngjs';
+import pixelmatch from 'pixelmatch';
+import { createHash } from 'crypto';
+
+const compare = (snapshot, latest, output, update, threshold = 0.01) => new Promise((resolve, reject) => {
+    const latestImg = PNG.sync.read(fs.readFileSync(latest));
+
+    if(update || !fs.existsSync(snapshot)) {
+        fsExtra.ensureFileSync(snapshot);
+        fs.writeFileSync(snapshot, PNG.sync.write(latestImg, { filterType: 4 }))
+        resolve({diff: 0, total: 0, msg: "Snapshot did not exist we created it for you!"})
```

Comment:
Similar comment about code style consistency, previous there was wrapping spaces `{ foo: 'bar' }` in objects literals.